OData/src/System.Web.OData/OData/Query/Expressions/FilterBinder.cs

```diff
@@ -32,6 +32,8 @@ internal class FilterBinder
     {
         private const string ODataItParameterName = "$it";
 
+        private static readonly string _dictionaryStringObjectIndexerName = typeof (Dictionary<string, object>).GetDefaultMembers()[0].Name;
```

Comment:
remove white space after "typeof"

please run "build.cmd" to make sure build with any error